tools/autograd/load_derivatives.py

```diff
@@ -422,7 +461,13 @@ def set_up_derivatives(f: NativeFunction) -> Tuple[
                            "but this is only allowed for outputs."
                            "Please use a different name in native_functions.yaml.")
 
-    derivatives, forward_derivatives, args_with_derivatives, non_differentiable_arg_names = set_up_derivatives(canonical)
+    (derivatives, forward_derivatives, args_with_derivatives,
```

Comment:
This line splitting is pretty hard to read...

Reply:
what do you think about this instead? this way we have a left side and a right side of the operator, despite it all spanning multiple lines.

Reply:
test